src/web3/oracle.ts

```diff
@@ -22,7 +22,7 @@ export async function genDeployAuthProof(
     let genAuthProof: HttpsCallable;
     if (identityType === "twitter.com" && authType === "oauth") {
```

Comment:
we can remove oauth check here as well. In the future we can remove auth type and use identity type only